api/src/paths/project/{projectId}.ts

```diff
@@ -82,11 +84,37 @@ function getProject(): RequestHandler {
 
       await connection.open();
 
-      const createResponse = await connection.query(getProjectSQLStatement.text, getProjectSQLStatement.values);
+      const [projectData, regionsData] = await Promise.all([
+        await connection.query(getProjectSQLStatement.text, getProjectSQLStatement.values),
+        await connection.query(getRegionsByProjectSQLStatement.text, getRegionsByProjectSQLStatement.values)
+      ]);
+
+      defaultLog.debug({ label: 'getProjectWithDetails', message: 'test' });
 
       await connection.commit();
 
-      const result = (createResponse && createResponse.rows && createResponse.rows[0]) || null;
+      connection.release();
+
+      await connection.commit();
```

Comment:
We only want to commit here once, and we don't need to release here as that gets called in the 'finally' block at the bottom already.